### ISSUE TITLE
packagedb-admin should not ask for EPEL branch before master branch

### ISSUE THREAD

**Issue by tyll** [fedora-infra/packagedb-cli#82]
packagedb-admin asked me to approve the epel7 branch of https://admin.fedoraproject.org/pkgdb/package/rpms/python3-Cython/timeline and later asked me to approve the master branch. Creating the master branch seems to create too many 500 errors:
```
RetryError                                Traceback (most recent call last)
/home/till/scm-genius/opensource/gh-tyll-packagedb-cli/pkgdb2client/admin.py in <module>()
    582 
    583 if __name__ == '__main__':
--> 584     main()

/home/till/scm-genius/opensource/gh-tyll-packagedb-cli/pkgdb2client/admin.py in main()
    549 
    550     try:
--> 551         arg.func(arg)
    552     except KeyboardInterrupt:
    553         print("\nInterrupted by user.")

/home/till/scm-genius/opensource/gh-tyll-packagedb-cli/pkgdb2client/admin.py in do_process(args)
    490 
    491         if action['action'] == 'request.package':
--> 492             data = __handle_request_package(actionid, action)
    493         elif action['action'] == 'request.branch':
    494             data = __handle_request_branch(actionid, action, package)

/home/till/scm-genius/opensource/gh-tyll-packagedb-cli/pkgdb2client/admin.py in __handle_request_package(actionid, action)
    304             namespace=action['info']['pkg_namespace'],
    305             monitoring_status=action['info'].get('monitoring_status', True),
--> 306             koschei=action['info'].get('koschei', False),
    307         )
    308 

/home/till/scm-genius/opensource/gh-tyll-packagedb-cli/pkgdb2client/__init__.pyc in create_package(self, pkgname, summary, description, review_url, status, shouldopen, branches, poc, upstream_url, critpath, namespace, monitoring_status, koschei)
    332             args['critpath'] = critpath
    333 
--> 334         return self.handle_api_call('/package/new/', data=args)
    335 
    336     def get_critpath_packages(self, branches=None, **kwargs):

/home/till/scm-genius/opensource/gh-tyll-packagedb-cli/pkgdb2client/__init__.pyc in handle_api_call(self, path, params, data)
    209         '''
    210 
--> 211         output = self.call_api(path, params, data)
    212 
    213         if not output or 'error' in output:

/home/till/scm-genius/opensource/gh-tyll-packagedb-cli/pkgdb2client/__init__.pyc in call_api(self, path, params, data)
    190             if not success:
    191                 raise PkgDBAuthException("Too many failed login attempts")
--> 192             data = self.send_request(url, **kwargs)
    193         return data
    194 

/usr/lib/python2.7/site-packages/fedora/client/openidbaseclient.pyc in send_request(self, method, auth, verb, **kwargs)
    238 
    239         try:
--> 240             output = func(method, **kwargs)
    241         except LoginRequiredError:
    242             raise AuthError()

/usr/lib/python2.7/site-packages/requests/sessions.pyc in post(self, url, data, json, **kwargs)
    509         """
    510 
--> 511         return self.request('POST', url, data=data, json=json, **kwargs)
    512 
    513     def put(self, url, data=None, **kwargs):

/usr/lib/python2.7/site-packages/requests/sessions.pyc in request(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)
    466         }
    467         send_kwargs.update(settings)
--> 468         resp = self.send(prep, **send_kwargs)
    469 
    470         return resp

/usr/lib/python2.7/site-packages/requests/sessions.pyc in send(self, request, **kwargs)
    574 
    575         # Send the request
--> 576         r = adapter.send(request, **kwargs)
    577 
    578         # Total elapsed time of the request (approximately)

/usr/lib/python2.7/site-packages/requests/adapters.pyc in send(self, request, stream, timeout, verify, cert, proxies)
    433 
    434             if isinstance(e.reason, ResponseError):
--> 435                 raise RetryError(e, request=request)
    436 
    437             raise ConnectionError(e, request=request)

RetryError: HTTPSConnectionPool(host='admin.fedoraproject.org', port=443): Max retries exceeded with url: /pkgdb/api/package/new/ (Caused by ResponseError('too many 500 error responses',))
> /usr/lib/python2.7/site-packages/requests/adapters.py(435)send()
    434             if isinstance(e.reason, ResponseError):
--> 435                 raise RetryError(e, request=request)
    436 

ipdb> e
MaxRetryError("HTTPSConnectionPool(host='admin.fedoraproject.org', port=443): Max retries exceeded with url: /pkgdb/api/package/new/ (Caused by ResponseError('too many 500 error responses',))",)
ipdb> u
> /usr/lib/python2.7/site-packages/requests/sessions.py(576)send()
    575         # Send the request
--> 576         r = adapter.send(request, **kwargs)
    577 

ipdb> u
> /usr/lib/python2.7/site-packages/requests/sessions.py(468)request()
    467         send_kwargs.update(settings)
--> 468         resp = self.send(prep, **send_kwargs)
    469 

```

I assume the error might be that the master branch should be created first. Maybe the server error logs reveal something.

**Comment by pypingou** [comment]
hm, I think the too many 500 errors was fixed by #92

**Comment by pypingou** [comment]
I'm going to close this ticket for now.

Let's re-open is it still happens, thanks for your report!
